deployer/run.sh

```diff
@@ -83,7 +83,7 @@ metric_resolution=${METRIC_RESOLUTION:-30s}
 project=${PROJECT:-openshift-infra}
 
 # the master certificate and service account tokens
-master_ca=${MASTER_CA:-/var/run/secrets/kubernetes.io/serviceaccount/ca.crt}
+master_ca=${MASTER_CA:-/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt}
```

Comment:
This is for connecting to the master api, which is signed with the main ca certificate and not the service ca certificate.